src/Controller/DashboardController.java

```diff
@@ -201,6 +269,12 @@ private void printToConsole(String output) {
 	private void updateDashboard() {
 		// reset name of logged in user
 		currentUser.setText(sim.getLoggedInUser().getName());
+		if (sim.getLoggedInUser().getLocation() == null) {
```

Comment:
When I test 'Current Location: ...' never changes for me.

Reply:
i realized this is because the label is too narrow.  It should be given a bigger minWidth or something.  So this is executing but we can never see the result.